docs/index.md

```diff
@@ -23,3 +23,23 @@ or pip:
 ```bash
 pip install graphium
 ```
+
+### IPU installation
+
+```bash
+mkdir ~/.venv                               # Create the folder for the environment
+python3 -m venv ~/.venv/graphium_ipu        # Create the environment
+source ~/.venv/graphium_ipu/bin/activate    # Activate the environment
+
+# Install the PopTorch wheel
+pip install PATH_TO_SDK/poptorch-3.2.0+109946_bb50ce43ab_ubuntu_20_04-cp38-cp38-linux_x86_64.whl
+
+# Enable Poplar SDK (including Poplar and PopART)
+source PATH_TO_SDK/enable
+
+# Install the IPU specific and graphium requirements
+PACKAGE_NAME=pytorch pip install -r requirements_ipu.txt
+
```

Comment:
Could we add this line for the lightning installation like here:
 https://github.com/datamol-io/graphium/blob/main/README.md?plain=1#L56